charts/portal/values-production.yaml

```diff
@@ -72,6 +72,13 @@ tls:
 # If you're migrating from a docker swarm environment and wish to keep your old internal certificates
 # then follow the instructions here to load your certificates into secrets manually #### Link to doc.
 # The job will still run, but will not update anything.
+# During helm upgrade, ensure to set this enabled value to 'false' which prevents from recreating the certificates depending on rotate.
+# If the certificates are expired, during helm upgrade set this value to 'true' to create new certificates. 
```

Comment:
The value should only be false if you intend not to run the job at all - tls.job.rotate determines upgrades.

Reply:
Included